CODEOWNERS

```diff
@@ -1 +1,2 @@
-* @c-parsons @laurentlb @jin
+* @c-parsons @laurentlb @jin @aiuto
+distribution @aiuto @fwe
```

Comment:
For my own information, what's the significance of "distribution changes"?

Reply:
https://help.github.com/en/articles/about-code-owners

I'm guessing you want `distribution/*` instead.

Reply:
Ah so this is a directory-to-be?

Reply:
Nvm. I see the other PR.

Reply:
Err, approved after this is changed to distribution/* :)

Reply:
Oh. duh. Yes. The empty directory shows up in my fork, so I don't notice it not being real yet.

Reply:
It doesn't need the trailing *, just the /